src/components/FreelancerEmptyProfile/FreelancerEmptyProfile.jsx

```diff
@@ -22,23 +23,23 @@ const FreelancerEmptyProfile = () => {
 
   const handleProfileChange = (newName, newRole) => {
     setFirstName(newName); // Update firstName state
-    setLastName(newRole); // Update lastName state
+    setRole(newRole); // Update lastName state
   };
 
   const handleSubmit = async () => {
     try {
       const payload = {
         password: "staticPassword",
```

Comment:
Can we remove this field? `password` should not be supported here. Backend has removed this.